testing/fstrm/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor:
```

Comment:
Please fill `Contributor` and `Maintainer` with your name and mail-address.

Reply:
Maintainer is only really needed when moving it to community or main.